Library/Formula/testssl.rb

```diff
@@ -0,0 +1,19 @@
+class Testssl < Formula
+  desc "Tool which checks for the support of TLS/SSL ciphers and flaws"
+  homepage "https://testssl.sh/"
+  url "https://github.com/jsarenik/testssl.sh/releases/download/v2.6/testssl-2.6.tar.gz"
+  version "2.6"
+  sha256 "6fb0daf68a8460ba48f8df0800093a1d4ef4fb9ecc923ea19416450c26977ba3"
+
+  depends_on "openssl"
+
+  def install
+    system "mkdir", "#{prefix}/bin"
+    system "cp", "mapping-rfc.txt", "#{prefix}/bin/mapping-rfc.txt"
+    system "cp", "testssl.sh", "#{prefix}/bin"
```

Comment:
You can kill off the `mkdir bin` line further up and just do `bin.install "testssl.sh"` here.